pages/docs/tools/asyncapi-cli.md

```diff
@@ -0,0 +1,137 @@
+---
+title: AsyncAPI CLI
+weight: 1
+---
+
```

Comment:
Let's add a Header & section for `Introduction`. We shouldn't go directly to installation instructions without introducing the tool.